SP/main.yaml

```diff
@@ -0,0 +1,122 @@
+sources:
+  IR_IMERG:
+    args:
+      consolidated: true
+    urlpath: /dados/hackathon/IMERG/{{yyyy}}/{{mm}}/*.nc
+    driver: NetCDF
+    parameters:
+      zoom:
+        allowed:
+          - 9
+        default: 9
+        description: zoom resolution of the dataset
+        type: int
+    metadata:
+      title: NCEP/CPC 4km Global (60N - 60S) IR and GPM IMERG (precipitation) Dataset
+      time_start: 2020-01-01T00:00:00
+      time_end: 2021-04-30T23:00:00
+      creator_name: Paola Bueno, Edson Barbosa
+      creator_email: paola.bueno@iag.usp.br, edson@meteoia.com
+      history: Obtained and processed by Paola Bueno, Edson Barbosa.
+      institution: IAG/USP
+  ERA5:
+    args:
+      consolidated: true
+      urlpath: /dados/hackathon/ERA5/{{level}}/*.nc
+    driver: NetCDF
+    parameters:
+      zoom:
+        allowed:
+          - 7
+        default: 7
+        description: zoom resolution of the dataset
+        type: int
+    metadata:
+      title: ERA5 reanalysis
+      time_start: 2020-01-01T00:00:00
+      time_end: 2021-04-30T23:00:00
+      creator_name: Paola Bueno, Edson Barbosa
+      creator_email: paola.bueno@iag.usp.br, edson@meteoia.com
+      history: Obtained and processed by Paola Bueno, Edson Barbosa.
+      institution: IAG/USP
+    summary: |  
+       Contains modified Copernicus Climate Change Service information 2020. Neither the European Commission nor ECMWF is responsible for any use that may be made of the Copernicus information or data it contains.
+       
+  ERA5Land:
+    args:
+      consolidated: true
+      urlpath: /dados/hackathon/ERA5Land/*.nc
+    driver: zarr
+    parameters:
+      zoom:
+        allowed:
+          - 7
+        default: 7
+        description: zoom resolution of the dataset
+        type: int
+    metadata:
+      title: 
+      time_start: 2020-01-01T00:00:00
+      time_end: 2021-04-30T23:00:00
+      creator_name: Paola Bueno, Edson Barbosa
+      creator_email: paola.bueno@iag.usp.br, edson@meteoia.com
+      history: Obtained and processed by Paola Bueno, Edson Barbosa.
+      institution: IAG/USP
+  um_glm_n2560_RAL3p3:
+    args:
+      chunks: null
+      consolidated: true
+      urlpath:
+        - /dados/hackathon/glm.n2560_RAL3p3/{{zoom}}.zarr
+    driver: zarr
+    parameters:
+      time:
+        allowed:
+        - PT1H
+        - PT3H
+        default: PT1H

```

Comment:
The `urlpath` of the dataset doesn't use the `time` parameter. So the returned dataset would be independent of this parameter, which is certainly wrong (the dataset is already available and different for the two options).
In case you have only one of the two options available, it would be necessary to restrict the list of `allowed` values to only the one you have locally available.
It would then be ok to not use the parameter in the `urlpath`, but usually you'd still have it somewhere used.